lib/routes/account.js

```diff
@@ -509,13 +510,35 @@ module.exports = function (
           if (shouldSendNewDeviceLoginEmail) {
             // The response doesn't have to wait for this,
             // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
-              userAgent.call({
-                acceptLanguage: request.app.acceptLanguage,
-                timestamp: Date.now()
-              }, request.headers['user-agent'])
-            )
+            var location = null
+            var timeZone = null
+            var ip = request.app.clientAddress
+            geodb(ip)
+              .then(
+                function (loc) {
+                  location = {
+                    city: loc.city,
+                    country: loc.country
+                  }
+                  timeZone = loc.timeZone
+                }, function (err) {
+                  // log error stuff
+                  console.log('im so sorry :(' + err.message)
```

Comment:
Looks like other sections do a something like this https://github.com/mozilla/fxa-auth-server/blob/656a6208a4e050add7d92d5a69ab07d074ec89e4/lib/routes/account.js#L275. I think it is reasonable to do something similar.